build/development-guide/smart-contracts/get-started-evm/use-hardhat.md

```diff
@@ -0,0 +1,96 @@
+# Using Hardhat to deploy on Acala, Karura & Mandala
+
+This example is inpired by the [Getting Started](https://hardhat.org/getting-started/) section of Hardhat Documentation.  
+
+## Prerequisites
+- node
+- yarn
+- npx
+
+
+## Let's create a basic sample project
+
+```bash=
+mkdir hardhat && cd hardhat
+yarn add hardhat
+npx hardhat
```

Comment:
we can replace all the `npx` command with `yarn` to be more consistent, since in the first step we use `yarn add hardhat`, which will create a `yarn.lock`. `npx` will use npm instead of `yarn`